ci/Jenkinsfile

```diff
@@ -43,7 +43,7 @@ pipeline {
                     }
                     agent {
                         docker {
-                            image 'gpuci/rapidsai:21.10-cuda11.4-devel-ubuntu20.04-py3.8'
+                            image 'gpuci/rapidsai:22.04-cuda11.5-devel-centos7-py3.8'
```

Comment:
Can we use env variables in the docker image? Would be better to have it as `image 'gpuci/rapidsai:${RAPIDS_VER}-cuda${CUDA_VER}-devel-centos7-py${PYTHON_VER}'`

Reply:
we don't change this often enough to matter, and there's a chance the image won't be available, so best to let people search for a good image and just copy paste.